esp-bootloader-esp-idf/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support reading partition tables and conveniently read/write partition content (#3316)
 
+- OTA-DATA partition support (#3354)
```

Comment:
Why are we tracking changes in a package that hasn't been released yet?

Reply:
my idea was to have the supported features here .... I don't insist on that 😆

Reply:
but I found it better than having the PRs always marked as "skip-changelog" 🤷‍♂️

Reply:
I don't mind this at all, we can just dump it in an "initial release" section.